types/aws-lambda/tslint.json

```diff
@@ -1,6 +1,7 @@
 {
     "extends": "dtslint/dt.json",
     "rules": {
-        "npm-naming": false
+        "npm-naming": false,
+        "void-return": false
```

Comment:
I had to remove this because I started getting this error:

```
Error: /Users/benoit/projects/DefinitelyTyped/types/aws-lambda/trigger/sqs.d.ts:3:63
ERROR: 3:63  void-return  Use the `void` type for return types only. Otherwise, use `undefined`. See: https://github.com/Microsoft/dtslint/blob/master/docs/void-return.md
```

I did not want to introduce any breaking change.